bundles/ui/org.eclipse.smarthome.ui/META-INF/MANIFEST.MF

```diff
@@ -18,6 +18,7 @@ Import-Package: javax.imageio,
  org.eclipse.jetty.client.api,
  org.eclipse.jetty.client.util,
  org.eclipse.jetty.http,
+ org.eclipse.jetty.proxy,
```

Comment:
@kaikreuzer Just to be clear, are you suggesting that I add `optional=true` now, and that no other code changes are required for this to work in the way you described?  Once org.eclipse.jetty.proxy is in the TP (which I think you are going to do), is `optional=true` needed?